packages/flutter/lib/src/cupertino/checkbox.dart

```diff
@@ -108,16 +124,69 @@ class CupertinoCheckbox extends StatefulWidget {
   /// ```
   final ValueChanged<bool?>? onChanged;
 
+  /// The cursor for a mouse pointer when it enters or is hovering over the
+  /// widget.
+  ///
+  /// If [mouseCursor] is a [WidgetStateProperty<MouseCursor>],
+  /// [WidgetStateProperty.resolve] is used for the following [WidgetState]s:
+  ///
+  ///  * [WidgetState.selected].
+  ///  * [WidgetState.hovered].
+  ///  * [WidgetState.focused].
+  ///  * [WidgetState.disabled].
+  ///
+  /// When [value] is null and [tristate] is true, [WidgetState.selected] is
+  /// included as a state.
+  ///
+  /// If null, then [SystemMouseCursors.basic] is used.
```

Comment:
Have you thought about using `@macro`s to reuse any of these docs between the two classes? Just wanted to make sure you thought about it, no need to insert macros everywhere, sometimes tons of macros is worse than duplicate comments.